app/public/js/services/firebaseService.js

```diff
@@ -1,4 +1,14 @@
-angular.module('FirebaseService', []).service('firebaseService', function($rootScope) {
+angular.module('FirebaseService', []).service('firebaseService', function($firebaseArray) {
+
+  this.getAll = function(key) {
+    var ref = firebase.database().ref().child(key);
```

Comment:
If you want, you can simply put the key inside the ref itself. Using child makes it more readable when you chain multiple keys, but isn't necessary. But yeah, as you must have tested, it does work both ways.